2.1.0/docker-entrypoint.sh

```diff
@@ -36,6 +36,9 @@ if [ "$1" = '/opt/couchdb/bin/couchdb' ]; then
 
 	if [ ! -z "$NODENAME" ] && ! grep "couchdb@" /opt/couchdb/etc/vm.args; then
 		echo "-name couchdb@$NODENAME" >> /opt/couchdb/etc/vm.args
+    else
+        NODENAME=$(hostname -f)
+        echo "-name couchdb@$NODENAME" >> /opt/couchdb/etc/vm.args
```

Comment:
Are you sure this logic is right? I'm not sure I understand the context but it looks like this stanza will write a duplicate `-name` line if `NODENAME` is non-empty but `couchdb@` was already in vm.args.

Maybe refactor to:
```
if ! grep "couchdb@" ...; then
  if [ -z "$NODENAME" ]; then
    NODENAME=$(hostname -f)
  fi
  echo "-name couchdb@$NODENAME" ...
fi